mongodb/spring-cloud-starter-stream-source-mongodb/README.adoc

```diff
@@ -0,0 +1,26 @@
+//tag::ref-doc[]
+= MongoDB Source
+
+This source polls data from MongoDB.
+This source is fully based on the `MongoDataAutoConfiguration`, so refer to the
+http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-nosql.html#boot-features-mongodb[Spring Boot MongoDB Support] for more
+information.
+
+== Options
+
+The **$$mongodb$$** $$source$$ has the following options:
+
+$$collection$$:: $$the collection to target$$ *($$String$$, no default, required)*
```

Comment:
"to target" for what or to what ?
Seems for me something isn't enough here :smile: